documentation/tutorial_setupProduction.md

```diff
@@ -36,9 +36,10 @@ Ensure that you are familiar with the following third-party technologies:<br>
    * `$CORTEX_URL` with Cortex server URL.
    * `$DOMAIN` with domain name without `http://`.
    * `$SSL_CERT_PATH` with the certificate file path in the remote server. <br>
-    For example, `/etc/letsencrypt/live/reference.elasticpath.com/fullchain.pem`.
+    For example, `/etc/letsencrypt/live/reference.elasticpath.com/fullchain.pem`. <br>
    * `$SSL_KEY_PATH` with the path of the private key in the remote server.<br>
-   For example, `/etc/letsencrypt/live/reference.elasticpath.com/privkey.pem`.
+   For example, `/etc/letsencrypt/live/reference.elasticpath.com/privkey.pem`. <br>
+   **Note:** You may use a certificate generated through any certificate provider of your choosing. An example is [Let's Encrypt](https://letsencrypt.org).
```

Comment:
You can use the certificate generated by any certificate provider, such as [Let's Encrypt](https://letsencrypt.org).